### ISSUE TITLE
Reload versions tab after editing/saving in editor

### ISSUE THREAD

**Issue by PVince81** [nextcloud/server#35510]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
When in the text editor and saving, if the versions tab is open it will not automatically reload the list.
One needs to close the sidebar then open it again.

**Describe the solution you'd like**
Technically, every editor should trigger a global Javascript event (on the event bus?) so that the sidebar tabs knows that a file has been edited. The versions tab can then listen to that event to reload itself.

**Describe alternatives you've considered**
N/A

**Additional context**

@juliushaertl are the text apps and office already triggering such events ?

cc @artonge @CarlSchwan as we discussed this as part of versions.

I suspect that implementing this is rather trivial, most time will likely been spent discussing the name of the event and mechanism :sweat_smile:

**Comment by juliushaertl** [comment]
Yep, I think we could just emit the same events that the viewer already uses:

https://github.com/nextcloud/server/issues/35511#issuecomment-1335405045

**Comment by juliushaertl** [comment]
Pushed a pull request for text to emit `emit('files:file:updated', { fileid: 123456 })` as viewer already does that for image editing: https://github.com/nextcloud/text/pull/3524

Collabora might be a bit trickier since we don't get a simple saved message from them, but we can track the state of the Doc_ModifiedStatus post message there and consider a save if the status changes from true to false. Will check the next days if that works properly and do a similar PR there then.
